### ISSUE TITLE
Overridden asGeoJson methods in OGCMultiLineString and OGCMultiPolygon drop spatial reference

### ISSUE THREAD

**Issue by sllynn** [Esri/geometry-api-java#290]
Hello Esri team and wider contributing group.

Both of the classes mentioned in the issue title have overrides to the default asGeoJson method implemented in OGCGeometry. On calling `op.execute` for the GeoJSON writer, both pass in a null in place of the spatial reference of the geometry. The GeoJson output then contains a `{"crs": null}` object.

Is this by design? It differs from the behaviour of e.g. ST_AsGeoJSON in PostGIS where we would see:

```
SELECT ST_AsGeoJSON(geom) FROM mytable;
                                                         st_asgeojson                                                         
------------------------------------------------------------------------------------------------------------------------------
 {"type":"MultiPolygon","crs":{"type":"name","properties":{"name":"EPSG:26910"}},"coordinates":[[[[0,0],[0,1],[2,2],[0,0]]]]}
```

Happy to open a PR if there's no good reason why the `esriSR` shouldn't be passed to the GeoJSON writer.

**Comment by stolstov** [comment]
@sllynn Yes, this looks like a typo. You can open a PR if you have a fix.

**Comment by sllynn** [comment]
I'll let you merge, since I don't have permission.

Do you have a new release planned?

**Comment by randallwhitman** [comment]
Geometry API for Java 2.3.0 is released .
